lib/internal/streams/end-of-stream.js

```diff
@@ -38,6 +38,16 @@ const {
   willEmitClose: _willEmitClose,
 } = require('internal/streams/utils');
 
+const {
+  isBrandCheck,
+  kState,
+} = require('internal/webstreams/util');
+
+const isReadableStream =
+  isBrandCheck('ReadableStream');
+const isWritableStream =
+  isBrandCheck('WritableStream');
```

Comment:
I don't think we can use isBrandCheck due to `readable-stream`. @mcollina?

Reply:
any alternatives? I had tried importing isReadableStream from `internal/webstreams/readablestream' but that makes the build fail

Reply:
Check for expected signature? For example check for pipeThrough.

Reply:
Done removed the brandchecks based checking